dev-tools/ecs-migration.yml

```diff
@@ -47,6 +47,28 @@
   alias6: true
   alias: true
 
+# Docker processor
+- from: docker.container.id
+  to: container.id
+  alias: false
+  alias6: true
+
+- from: docker.container.image
+  to: container.image.name
+  alias6: true
+  alias: true
+
+- from: docker.container.name
+  to: container.name
+  alias6: true
+  alias: true
+
+- from: docker.container.labels  # TODO: How to map these?
```

Comment:
@graphaelli Have you found a good way to migrate such fields?

Reply:
I don't think we can. I'm assuming this is user-populated, including key names? If that's the case, then the list of fields to migrate is unknowable.

Perhaps if some fields are often present by default, we can create aliases for those, though.

Reply:
I have not found a good way.  Can the migration assistant create those aliases dynamically?  That is, iterate over the source labels and create aliases on the old indices for each one?  Another option is to inform users to include both fields with an or clause in searches and aggregations.

APM has the same issue  where `context.tags` are moving to `labels`.  It may be possible there to introduce an APM UI-only concept of "tags" and make the `context.tags.$key=$value or labels.tags.$key=value` query on behalf of the user - we have not discussed this possibility but I'd be curious what you think.

Reply:
Yeah I think the migration assistant will be our best bet.

Reply:
@graphaelli Can you take this up with the Kibana team working on the migration assistant to see what they think?

Reply:
They've added it to their use cases for the migration assistant, there will be hooks for us to handle it there.

Reply:
Great news. Thanks for following up.